### PR TITLE
docs: update introduction

### PR DESCRIPTION
**Pull request by ahhshm**
- renamed "Introducing tRPC​" into "Features". You already introduced tRPC in the first paragraph.
- synced the features section with README

### CODE REVIEW

File: www/docs/main/introduction.md [hunk 1]
```diff
@@ -30,29 +30,22 @@ author_image_url: https://avatars1.githubusercontent.com/u/459267?s=460&v=4
 
 ## Introduction
 
-<abbr title="TypeScript Remote Procedure Call">tRPC</abbr> allows you to easily build & consume fully typesafe APIs, without schemas or code generation.
```
Comment (KATT) [inline www/docs/main/introduction.md:33]: I'd like to keep this

**Review by KATT — CHANGES_REQUESTED** [review]
Please address the failing build 🙏 

If you setup github sponsors I'm going to send you a token of gratitude :)

File: www/docs/main/introduction.md (hunk 1 above)
Comment (ahhshm) [reply www/docs/main/introduction.md:33]: Done!

**Comment by ahhshm** [comment]
Thank you so much. I really appreciate it but unfortunately, due to the US sanctions on Iran, my country's banks can't connect to the international bank system and I can't have a Paypal account so setting up github sponsors isn't possible for me.

File: www/docs/main/introduction.md (hunk 1 above)
Comment (ahhshm) [reply www/docs/main/introduction.md:33]: Done

**Review by KATT — APPROVED** [review]

**Pull request merged**